pkg/storaged/side-panel.jsx

```diff
@@ -134,7 +134,7 @@ class SidePanelRow extends React.Component {
                     {this.props.actions}
                 </div>);
         else if (client.path_jobs[job_path])
-            decoration = <Spinner isSVG size="md" />;
+            decoration = <Spinner size="md" />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)